subprojects/core/src/integTest/groovy/org/gradle/api/ApplyPluginIntegSpec.groovy

```diff
@@ -28,7 +28,9 @@ import static org.gradle.util.internal.TextUtil.normaliseFileSeparators
 
 // TODO: This needs a better home - Possibly in the test kit package in the future
 @Issue("https://github.com/gradle/gradle-private/issues/3247")
-@IgnoreIf({ OperatingSystem.current().macOsX && JavaVersion.current() == JavaVersion.VERSION_1_8})
+@IgnoreIf({ OperatingSystem.current().macOsX && JavaVersion.current() == JavaVersion.VERSION_1_8 })
+// The gradleApi() dependency has missing JARs unless the test is run with the full Gradle distribution
+@IgnoreIf({ GradleContextualExecuter.embedded })
```

Comment:
Just curious: will CI always run this with the `forking` and `configCache` executors?

Reply:
I am not sure what you mean by this, exactly? Our pipeline does always run non-embedded executors before a merge occurs.